docs/ivys.js

```diff
@@ -16,6 +16,54 @@ function IvyApp($grid) {
     this.$menuArrow.click(this.toggleFooterMenu);
 }
 
+IvyApp.prototype.initFirebase = function () {
+    "use strict";
+    // Shortcuts to Firebase SDK features.
+    this.auth = firebase.auth();
+    this.storage = firebase.storage();
+    this.storageRef = this.storage.ref();
+    
+    // Logs debugging information to the console.
+    firebase.database.enableLogging(false);
+    
+    // Initiates Firebase auth and listen to auth state changes.
+    this.auth.onAuthStateChanged(this.onAuthStateChanged.bind(this));
+
+};
+
+IvyApp.prototype.signIn = function () {
+    // Sign in Firebase using popup auth and Google as the identity provider.
+    "use strict";
+    
+    this.auth.signInAnonymously().catch(function (error) {
```

Comment:
Sorry, the comment is incorrect. Also, made it so that user is signed in when page loads.